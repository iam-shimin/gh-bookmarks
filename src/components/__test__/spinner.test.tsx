import React from 'react';
import { render } from '@testing-library/react';
import Spinner from '../spinner';

test('<Spinner />', () => {
	const { asFragment } = render(<Spinner />);
	expect(asFragment()).toMatchSnapshot();
});