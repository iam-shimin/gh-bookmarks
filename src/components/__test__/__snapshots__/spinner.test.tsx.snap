// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Spinner /> 1`] = `
<DocumentFragment>
  <div
    style="position: fixed; top: 50%; left: 50%; transform: translate(-50%, -50%);"
  >
    <div
      class="spinner-big spinner-border"
    />
  </div>
</DocumentFragment>
`;
